fastpay/src/client.rs

```diff
@@ -528,10 +529,17 @@ fn main() {
             initial_funding,
             num,
         } => {
-            let num_accounts: u32 = num;
-            for _ in 0..num_accounts {
-                let account = UserAccount::new(initial_funding);
-                println!("{}:{}", encode_address(&account.address), initial_funding);
+            for i in 0..num {
+                let account = UserAccount::new(
+                    AccountId::new(vec![SequenceNumber::from(i as u64)]),
+                    initial_funding,
+                );
+                println!(
```

Comment:
```suggestion
                info!(
```

Reply:
Actually, this one is meant to print in stdout.